.env_sample

```diff
@@ -1,19 +1,64 @@
-SUNLIGHT_API_KEY=""
-GOOGLE_MAP_API_KEY=<google_map_key>
-DJANGO_SECRET_KEY=<django_secret_key>
-PROPUBLICA_API_KEY=<pro_publica_api_key>
 
+DEBUG=on
+
+
+# database variables
+
+DATABASE_URL=postgresql://urser:un-githubbedpassword@127.0.0.1:8458/database
```

Comment:
We should use environment variables for the database configuration. Use f-strings to combine the `POSTGRES_HOST`, `POSTGRES_PORT`, `POSTGRES_DB`, `POSTGRES_USER`, and `POSTGRES_PASSWORD` into a single string.

Reply:
I didn't get this are you talking about settings file or env file?

Reply:
And I don't get this pep 8 thing what's that?

Reply:
Sorry for the delay. Pep8 is a code linter that makes sure we follow basic code conventions.